compiler/rustc_error_codes/src/error_codes/E0120.md

```diff
@@ -11,7 +11,15 @@ impl Drop for MyTrait {
 }
 ```
 
-A workaround for this problem is to wrap the trait up in a struct, and implement
+```compile_fail,E0120
+struct Concrete {}
+
+impl Drop for &'_ mut Concrete  {
+    fn drop(&mut self) {}
+}
+```
+
+A workaround for traits is to wrap the trait up in a struct, and implement
 Drop on that:
```

Comment:
I think this could be a bit more exact:

```suggestion
A workaround for traits is to create a wrapper struct with a generic type,
add a trait bound to the type, and implement `Drop` on the wrapper:
```